src/services/pocket-relayer.ts

```diff
@@ -440,6 +441,24 @@ export class PocketRelayer {
       let nodes: Node[] = pocketSession.sessionNodes
       const relayStart = process.hrtime()
 
+      const cachedRemovedSessionNodes = await this.redis.get(`session-${pocketSession.sessionKey}`)
```

Comment:
This is a good trick. I forgot pocket-js already had the concept of a sessionKey.